composite/composite.go

```diff
@@ -48,10 +48,8 @@ func (c Compositor) Process(inputZipPath string, outputImagePath string) error {
 	return nil
 }
 
-func (c Compositor) savePng(image *image.NRGBA, outputPath string) {
-	buf := new(bytes.Buffer)
-	png.Encode(buf, image)
-	c.Storage.Write(outputPath, buf.Bytes())
+func (c Compositor) savePng(image image.Image, outputPath string) {
+	imagelib.Write(outputPath, image)
```

Comment:
I'm confused. This only uses `png.Encode` too. https://github.com/xyproto/imagelib/blob/bbbf5649f300ed19ce19b11d4cf8c333b09a53b8/readwrite.go#L39

The only difference I can see (to our original implementation) is that it is writing directly to a file, not a buffer first.

As this library is no longer maintained could you try writing to a file directly and see if that produces the same result.

Reply:
Good point didn't look into the library. I moved the code out of it and it still seems to produce images with the right size.  🎉